addons/account/models/account_move.py

```diff
@@ -1449,27 +1454,36 @@ def _compute_partner_credit_warning(self):
                            move.move_type == 'out_invoice' and \
                            move.company_id.account_use_credit_limit
             if show_warning:
+                total_field = 'amount_total' if move.currency_id == move.company_currency_id else 'amount_total_company_currency'
+                current_amount = move.tax_totals[total_field]
```

Comment:
```suggestion
                current_amount = move.amount_total_signed * move.direction_sign
```
Why do you use the tax_totals here? :sweat_smile:

Reply:
As discussed: We need to updated the warning during edit and the (correct) tax lines do not exist yet. Added the company currency version to the `tax_totals`.
There was nothing done for the cash rounding for the company currency since currently nothing is done for the cash in company currency